### DISCUSSION TITLE
Meal diagram

### DISCUSSION THREAD

**Discussion by gman193** [ccci-code/summerproject#14] [Ideas]
https://wireframe.corp.ccc-i.com/cbg30hskj5ehveoqtm9g

Here is my design for my meal diagram.

**Comment by gman193** [comment]
I'm having technical difficulties right now. Sorry that the link doesn't work.

**Reply by ccci-code** [reply]
Yes, it looks like it chopped of part of the URL.  I'll test one to see if I have the same problem:

[Test link](https://wireframe.corp.ccc-i.com/732cb23e-b0d4-afff-05bd-c7d7db676461)

https://wireframe.corp.ccc-i.com/732cb23e-b0d4-afff-05bd-c7d7db676461

You should be able to copy and paste the url in the text box.  Let me know if you still have problems.

**Reply by gman193** [reply]
https://wireframe.corp.ccc-i.com/cbg30hskj5ehveoqtm9g

I have no way of changing the URL, regardless of how many times I save the assignment.

**Reply by ccci-code** [reply]
> https://wireframe.corp.ccc-i.com/cbg30hskj5ehveoqtm9g
> 
> I have no way of changing the URL, regardless of how many times I save the assignment.

That's strange.  Do you still have the page open with the design?  If so, if you refresh the page, does it show a different URL?  The last part of the URL doesn't look right since it should have dashes included in it like this:  732cb23e-b0d4-afff-05bd-c7d7db676461

**Reply by gman193** [reply]
When I refresh the page, an error screen shows up.

**Reply by ccci-code** [reply]
I haven't figured out why some diagrams won't reload.  Can everyone take a screenshot of their diagram too?  You can attach it to your comment.

**Comment by gman193** [comment]
![mydraft page 1](https://user-images.githubusercontent.com/105887154/181099364-67b5cd84-2891-449e-92e8-e43c214e2289.PNG)
![mydraft page 2](https://user-images.githubusercontent.com/105887154/181099396-8ac8cf48-84de-4688-8f90-c089d2b6b730.PNG)

Here is my diagram idea.

**Reply by ccci-code** [reply]
Looks great!  It helps to show visually what you are thinking.  For the second diagram, what do you think about showing the results in a calendar type layout?  Maybe it could look similar to the way schools display their lunches.  Each meal would be in a box for that day.
